packages/typespec-ts/src/modular/buildCodeModel.ts

```diff
@@ -1589,25 +1589,10 @@ function addHierarchyOperationGroup(
   return [];
 }
 
-function resolveConflictIfExist(operationGroups: OperationGroup[]) {
+function resolveConflict(operationGroups: OperationGroup[]) {
```

Comment:
i was wondering if we could rename this as sth e.g appendOperationGroupPrefix to make it clearer because this is not mainly for resolving conflict now.